llvm/test/tools/sycl-post-link/emit_imported_symbols.ll

```diff
@@ -0,0 +1,67 @@
+; This test checks that the -emit-imported-symbols option generates a list of imported symbols
+;
+; RUN: sycl-post-link -symbols -emit-imported-symbols -split=kernel -S < %s -o %t.table
```

Comment:
Can we add testing for module split (I think it boils down to no split for this case) also? Thanks

Reply:
Added testing with -split=source.